QualityControl/test/public/pages/layout-show.test.js

```diff
@@ -200,7 +200,16 @@ export const layoutShowTests = async (url, page, timeout = 5000, testParent) =>
     'should have number input field for allowing users to change auto-tab value',
     { timeout },
     async () => {
-      await page.waitForSelector('#inputDescription', { timeout: 5000 });
+      const inputSelector = '#inputChangeTabTimer';
+      await page.waitForSelector(inputSelector, { timeout: 5000 });
+      await page.locator(inputSelector).fill('9');
```

Comment:
I believe `locator` already `waitsForSelector` or has the option to set a timeout to wait for it. Please have a look at the docs as you are duplicating functionality in this manner: https://pptr.dev/api/puppeteer.page.locator